lib/ApiRequestor.php

```diff
@@ -260,7 +260,8 @@ private static function _defaultHeaders($apiKey, $clientInfo = null)
         $uaString = 'Stripe/v1 PhpBindings/' . Stripe::VERSION;
 
         $langVersion = phpversion();
-        $uname = php_uname();
+        $uname_disabled = in_array('php_uname', explode(',', ini_get('disable_functions')));
+        $uname = $uname_disabled ? '(disabled)' : php_uname();
```

Comment:
Minor but have we considered making that call once when the library or client is initialized and then use a cached value instead of doing that on each request?

Reply:
I'll probably do just that if/when I ever get to rewriting the HTTP client layer.